app/assets/stylesheets/active_admin/_forms.scss

```diff
@@ -134,6 +141,10 @@ form {
     }
   }
 
+  input[type=date] {
+    width: calc(100% - #{$text-input-total-padding});
+  }
```

Comment:
@timoschilling Not sure why it was necessary -- The date input was visibility smaller than the other inputs so it was a quick fix. I can look into the underlying cause.